pkg/cli/subcommands/dcaconfigcheck/command.go

```diff
@@ -60,5 +63,13 @@ func MakeCommand(globalParamsGetter func() GlobalParams) *cobra.Command {
 }
 
 func run(_ log.Component, _ config.Component, cliParams *cliParams) error {
-	return flare.GetClusterAgentConfigCheck(color.Output, cliParams.verbose)
+	var b bytes.Buffer
+	color.Output = &b
```

Comment:
Not sure if the color part is correct. The output shows some colors in 7.54.0, but with a custom image built from this branch there are no colors.

Reply:
@davidor I fixed on this commit https://github.com/DataDog/datadog-agent/pull/26435/commits/58a34dc5a17e93ba1d68ba895cfa8007d22f5329. Let me know what you think 😄